### ISSUE TITLE
Text Width adjustement

### ISSUE THREAD

**Issue by Yattatux** [giuspen/cherrytree#2425]
Hello Giuspen, thanks for your great work.

I'm asking for a function to get the text centered on the edge of the Cherrytree window. In other words get some lateral empty space when the window is maximized (paragraph width). This feature could help when writing text. The same result is achieved with Cherrytree window resizing, but I wish it was possible with the maximized window.

Best Regards.

**Comment by giuspen** [comment]
You mean to be able to configure a left margin on the text?

**Comment by Yattatux** [comment]
Hello, I mean to be able to configure left and right margin too.

**Comment by giuspen** [comment]
I added the possibility to configure the left and right margins in the config.cfg only for now.
Can you test building yourself or would you like a binary?

**Comment by giuspen** [comment]
Yes, please find https://www.giuspen.net/software/CherryTree-1.0.4+20-x86_64.AppImage

**Comment by cwinsnes** [comment]
I have had the same issue and the left/right margins in config.cfg works well! Thank you for implementing such a thing!

An idea I have for making it easier to use is for the values to be able to be set using fractions instead. For example left margin at 10% of the width and right margin at 70% to make it adaptable for different screen sizes and such.
